### PR TITLE
Shrkanch external pt

### PR DESCRIPTION
**Pull request by shrkanchExternal**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/publish-codebuild.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#!/bin/bash -e
```
Comment (awsluja) [inline .circleci/publish-codebuild.sh:1]: commenting here for test

**Review by awsluja — CHANGES_REQUESTED** [review]
requesting some changes. - test

File: .circleci/publish-codebuild.sh (hunk 1 above)
Comment (shrkanchExternal) [reply .circleci/publish-codebuild.sh:1]: Thanks.

**Comment by shrkanchExternal** [comment]
Hi,

**Review by shrkanchExternal — COMMENTED** [review]
this is test.

**Review by shrkanchExternal — COMMENTED** [review]
this is for testing purpose only.

**Comment by shrkanchExternal** [comment]
`"'>

**Pull request closed without merging**